buildpack/telemetry/appdynamics.py

```diff
@@ -1,24 +1,36 @@
 import logging
 import os
+import subprocess
 
 from buildpack import util
 
-APPDYNAMICS_VERSION = "21.11.1.33280"
+APPDYNAMICS_VERSION = "22.1.0.33445"
+
+AGENT_PATH = os.path.join(os.path.abspath(".local"), "machineagent", "bin", "machine-agent")
 
 
 def stage(buildpack_dir, destination_path, cache_path):
     if appdynamics_used():
         util.resolve_dependency(
             util.get_blobstore_url(
                 "/mx-buildpack/appdynamics/appdynamics-agent-1.8-{}.zip".format(
-                    APPDYNAMICS_VERSION + "-mendix"
+                    APPDYNAMICS_VERSION
                 )
             ),
             destination_path,  # DOT_LOCAL_LOCATION,
             buildpack_dir=buildpack_dir,
             cache_dir=cache_path,  # CACHE_DIR,
         )
 
+        util.resolve_dependency(
+            util.get_blobstore_url(
+                "/mx-buildpack/appdynamics/appdynamics-machineagent-bundle-22.2.0.3282.zip"
```

Comment:
Can we extract out the machine agent version info and make it a global constant ?

Reply:
+